e3sm_diags/derivations/acme.py

```diff
@@ -2074,3 +2074,392 @@ def cosp_histogram_standardize(cld: "FileVariable"):
     "sitemptop": OrderedDict([(("sitemptop",), rename)]),
     "siv": OrderedDict([(("siv",), rename)]),
 }
+
+# Names of 2D aerosol burdens, including cloud-borne aerosols
+aero_burden_list = [
+    "ABURDENDUST",
+    "ABURDENSO4",
+    "ABURDENSO4_STR",
+    "ABURDENSO4_TRO",
+    "ABURDENPOM",
+    "ABURDENMOM",
+    "ABURDENSOA",
+    "ABURDENBC",
+    "ABURDENSEASALT",
+]
+
+
+def aero_burden_fxn(var):
+    """
+    Scale the aerosol burden by 1e6.
+
+    Parameters:
+        var (float): The input burden in kg/m2.
```

Comment:
```suggestion
    Parameters:
        var (cdms2.TransientVariable): The input burden in kg/m2.
```

I believe these are all `cdms2.TransientVariable` based on setting the `.units` attribute.